File: app/src/main/java/com/semye/android/jsbridge/BridgeHandler.java
package com.semye.android.jsbridge;

/**
 * bridge handler 接口
 */
public interface BridgeHandler {

    /**
     * 处理数据
     *
     * @param data     数据
     * @param function 回调函数
     */
    void handler(String data, CallBackFunction function);

}

File: app/src/main/java/com/semye/android/webview/jsbridge/BridgeHandler.kt
package com.semye.android.webview.jsbridge

/**
 * bridge handler 接口
 */
interface BridgeHandler {
    /**
     * 处理数据
     *
     * @param data     数据
     * @param function 回调函数
     */
    fun handler(data: String?, function: CallBackFunction?)
}